### PR TITLE
Fix logging

### PR DESCRIPTION
**Pull request by digeff**
We weren't using the logging level of the trace parameter

### CODE REVIEW

File: src/chrome/client/chromeDebugAdapter/unconnectedCDA.ts [hunk 1]
```diff
@@ -60,11 +60,22 @@ export class UnconnectedCDA implements IDebugAdapterState {
     private parseLoggingConfiguration(args: ILaunchRequestArgs | IAttachRequestArgs): ILoggingConfiguration {
         let traceValue: Logger.LogLevel;
         switch (args.trace) {
+            case true:
+            case 'true':
```
Comment (roblourens) [inline src/chrome/client/chromeDebugAdapter/unconnectedCDA.ts]: Why string `"true"`?

Comment (digeff) [reply src/chrome/client/chromeDebugAdapter/unconnectedCDA.ts]: In case someone mistakenly sends "true" instead of true.

Comment (roblourens) [reply src/chrome/client/chromeDebugAdapter/unconnectedCDA.ts]: Well they should not do that...

Comment (digeff) [reply src/chrome/client/chromeDebugAdapter/unconnectedCDA.ts]: Removing it.

**Pull request merged**